src/app/Login/page.tsx

```diff
@@ -0,0 +1,97 @@
+'use client';
+import React, { useState } from 'react';
+import ActiveButton from '@/components/Active-Button/ActiveButton';
+import InactiveButton from '@/components/Inactive-Button/InactiveButton';
+import InputField from '@/components/Input-textbox/textbox';
+import styles from './page.module.css';
+import { emailRegex } from '@/utils/constants';
+
+const isValidEmail = (email: string) => {
+  return emailRegex.test(email);
+};
```

Comment:
this function is not redundant since you are just returning another function call, this can be removed and wherever you call the function, just replace it with this

`emailRegex.test(email)`